src/gulp-jsdoc.js

```diff
@@ -59,6 +59,14 @@ export function jsdoc(config, done) {
                 reject(new Error(errMsg));
             }
 
+            if (!jsdocConfig.source) {
+                jsdocConfig.source = {};
```

Comment:
The `source` object allows multiple ways of specifying what files to use (`include`, `exclude`, `includePattern`, `excludePattern`). In my opinion, if any of these are set (i.e. the `source` object exists), then we should respect it.